package.json

```diff
@@ -1,10 +1,31 @@
 {
+  "name": "wikibase-data-model",
```

Comment:
I think we should stick to tab-indention in all .json files. But this can be fixed later.